test/test_runner_battle.c

```diff
@@ -1139,6 +1139,11 @@ void TestRunner_Battle_RecordExp(u32 battlerId, u32 oldExp, u32 newExp)
     }
 }
 
+#define IS_SKIPOVER_CHAR(char)  (char == CHAR_SPACE \
+                              || char == CHAR_PROMPT_SCROLL \
+                              || char == CHAR_PROMPT_CLEAR \
+                              || char == CHAR_NEWLINE)
+
```

Comment:
We should prefer `static inline` functions to macros wherever possible.
```suggestion
static inline bool32 IsSkipoverChar(u32 char)
{
  return char == CHAR_SPACE
      || char == CHAR_PROMPT_SCROLL
      || char == CHAR_PROMPT_CLEAR
      || char == CHAR_NEWLINE;
}
```